build/debian/mixxx.menu.in

```diff
@@ -0,0 +1,7 @@
+?package(mixxx): \
+  needs="X11" \
+  section="Apps/Sound" \
+  title="Mixxx" \
+  command="/usr/bin/pasuspender -- /usr/bin/mixxx -platform xcb" \
```

Comment:
```suggestion
  command="/usr/bin/pasuspender -- /usr/bin/mixxx -platform xcb || /usr/bin/mixxx -platform xcb" \
```
This should not require PulseAudio to be installed.

Reply:
I don't think is actually runs a full shell. And this automatically restarts mixxxx if it crashes or exists with non-zero status, which is confusing.

If pasuspender is installed by default on Ubuntu, I think it's okay to use it. If you really want the fallback, we should use a separate shell script that tests if that command is in `$PATH`.